config/locales/views/layouts/en.yml

```diff
@@ -27,7 +27,7 @@ en:
       language: "Language"
     mailer:
       join_us: "Join Us"
-      copyright_text: "Copyright © 2021 CircuitVerse. All rights reserved. For any issues, reach out to us support@circuitverse.org"
+      copyright_text: "Copyright © %{time_current_year} CircuitVerse, All rights reserved."
```

Comment:
You may have to update the mailer layout to make this work, also don't remove extra text